Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.19-alpine
```

Comment:
Esta versión ni siquiera es la última. También tenéis que justificar las versiones.

Reply:
Es la versión con la que he desarrollado el proyecto, ahora está la 1.20 también, que la han tenido que sacar recientemente o por lo menos cuando me instalé go no estaba todavía, voy a probar la nueva y si no hay problema con las dependencias, la actualizo

Reply:
la elección del contenedor base no tiene nada que ver con la que hayas usado para desarrollar el proyecto. Se trata de que lo pruebes para diferentes versiones.

Reply:
Entiendo, he probado la última versión y funciona :ok_hand: